modules/examples/todolist/client/src/main/scala/ClientProgram.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package examples.todolist.client
+
+import cats.Monad
+import cats.syntax.functor._
+import freestyle.tagless._
+import org.log4s.getLogger
+
+@tagless(true)
+trait PingPongClient[F[_]] {
+  def ping(): F[Unit]
+}
+
+object ClientProgram {
+
+  val logger = getLogger
+
+  def clientProgram[M[_]: Monad](implicit client: PingPongClient[M]): M[Unit] = {
+    for {
+      _ <- client.ping()
```

Comment:
If you only have one line in a for comprehension you can replace it like here https://docs.scala-lang.org/tutorials/FAQ/yield.html

Reply:
The for comprehension was totally unnecessary, good point. I remove it.